Below threshold Trace Analytics monitors (#13379)
DataDog/documentation: content/en/monitors/create/types/apm.md

To create an [APM monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> APM*.

> Choose between an **APM Metrics** or **Trace Analytics** monitor:

Reviewer: ```suggestion
Choose between an **APM Metrics** or a **Trace Analytics** monitor:
```

### Select alert conditions

> * Trigger when the query result is `above`, `above or equal to`, `below`, or `below or equal to`

Reviewer: ```suggestion
* Trigger when the query result is `above`, `above or equal to`, `below`, or `below or equal to`.
```

> * the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 5 minutes and 48 hours.

Reviewer: ```suggestion
* The threshold during the last `5 minutes`, `15 minutes`, `1 hour`, or `custom` to set a value between 5 minutes and 48 hours.
```

* Alert threshold: `<NUMBER>`
* Warning threshold: `<NUMBER>`

> #### No Data and below alerts

Reviewer: ```suggestion
#### No data and below alerts
```

> #### No data and below alerts

Reviewer 2: 1. Why do you think we should add this extra section? Don't you think it's expressed by the above section already?
2. If we decide to keep it, we might want to update it a little so that it is not only geared at multi alert monitors - WDYT?

> To receive a notification when a group matching a specific query have stopped sending spans, set the condition to below `1`. This notifies when no spans match the monitor query in the defined evaluation period for the group.

Reviewer: ```suggestion
To receive a notification when a group matching a specific query stops sending spans, set the condition to below `1`. This notifies you when no spans match the monitor query in the defined evaluation period for the group.
```
